File: rename_video.js
const fs = require('fs');
const path = require('path');

const videoDir = path.join(__dirname, 'src', 'assets', 'video');
const files = fs.readdirSync(videoDir);

files.forEach(file => {
  if (file.endsWith('.mp4') && file !== 'hero-1.mp4' && file !== 'hero-2.mp4') {
    const oldPath = path.join(videoDir, file);
    const newPath = path.join(videoDir, 'hero-1.mp4');
    try {
      fs.renameSync(oldPath, newPath);
      console.log(`Renamed ${file} to hero-1.mp4`);
    } catch (err) {
      console.error(`Error renaming ${file}:`, err);
    }
  }
});

File: rename_video.py
import os
import glob

video_dir = r"d:\codesinc projects\cunstruction\cunstruction-LTD\src\assets\video"
mp4_files = glob.glob(os.path.join(video_dir, "*.mp4"))

for f in mp4_files:
    if "hero-1.mp4" not in f:
        new_path = os.path.join(video_dir, "hero-1.mp4")
        try:
            os.rename(f, new_path)
            print(f"Renamed {f} to {new_path}")
        except Exception as e:
            print(f"Error renaming {f}: {e}")
